modules/gathering-data-specific-features.adoc

```diff
@@ -28,10 +28,10 @@ ifndef::openshift-origin[]
 |`registry.redhat.io/container-native-virtualization/cnv-must-gather-rhel8:v{HCOVersion}`
 |Data collection for {VirtProductName}.
 
-|`registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8`
+|`registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8:2.3`
```

Comment:
This is not right, I think. This image is not OSSM.
Here's the list of tags, but I have no idea which one you should use here. https://catalog.redhat.com/software/containers/openshift-serverless-1/svls-must-gather-rhel8/5e8ec10d5a134668769bde5f

Reply:
@jwendell Is that section for the CNV product ? I see "Data collection for {VirtProductName}."  above.
We only need to modify our OSSM product section must-gather.

Reply:
@yxun and @jwendell this was my mistake. I searched on istio-must-gather-rhel8, saw it in two files, and made the change. I will revert the change in the PR.